log.js

```diff
@@ -62,11 +63,12 @@ module.exports = function (dir, config, privateIndex, db) {
     let recBuffers = []
     let kvts = []
 
+    const timestamp = newTimestamp()
```

Comment:
This means that all msgs in the transaction would have the same timestamp. Is that okay/expected?

Reply:
They need to be unique. Right now we only use this function for indexes where the 2 mesages are on different feeds, but in general if you add 2 messages to the same feed, then validate will complain.